be/src/olap/delta_writer.cpp

```diff
@@ -128,6 +128,10 @@ OLAPStatus DeltaWriter::init() {
     writer_context.partition_id = _req.partition_id;
     writer_context.tablet_schema_hash = _req.schema_hash;
     writer_context.rowset_type = _storage_engine->default_rowset_type();
+    if (_tablet->tablet_meta()->has_preferred_rowset_type()
```

Comment:
Is it better to encapsulate it in a function?

Reply:
ok